docs/speech.rst

```diff
@@ -55,23 +54,15 @@ Functions
 
     As with the music module, you can use an optional argument to specify the 
     output pin can be used to override the default of ``microbit.pin0``. 
```

Comment:
I believe the pin is only configurable in v2?

Reply:
Updated